src/main/kotlin/net/leanix/githubagent/services/WebhookEventService.kt

```diff
@@ -47,39 +49,67 @@ class WebhookEventService(
         }
 
         if (pushEventPayload.ref == "refs/heads/${pushEventPayload.repository.defaultBranch}") {
-            when {
-                MANIFEST_FILE_NAME in headCommit.added -> {
-                    logger.info("Manifest file added to repository $repositoryFullName")
-                    val fileContent = getManifestFileContent(organizationName, repositoryName, installationToken)
-                    sendManifestData(repositoryFullName, ManifestFileAction.ADDED, fileContent)
-                }
-                MANIFEST_FILE_NAME in headCommit.modified -> {
-                    logger.info("Manifest file modified in repository $repositoryFullName")
-                    val fileContent = getManifestFileContent(organizationName, repositoryName, installationToken)
-                    sendManifestData(repositoryFullName, ManifestFileAction.MODIFIED, fileContent)
-                }
-                MANIFEST_FILE_NAME in headCommit.removed -> {
-                    logger.info("Manifest file removed from repository $repositoryFullName")
-                    sendManifestData(repositoryFullName, ManifestFileAction.REMOVED, null)
-                }
-            }
+            handleManifestFileChanges(
+                "${gitHubEnterpriseProperties.manifestFileDirectory}${ManifestFileName.YAML.fileName}",
+                headCommit,
+                repositoryFullName,
+                organizationName,
+                repositoryName,
+                installationToken
+            )
+            handleManifestFileChanges(
+                "${gitHubEnterpriseProperties.manifestFileDirectory}${ManifestFileName.YML.fileName}",
+                headCommit,
+                repositoryFullName,
+                organizationName,
+                repositoryName,
+                installationToken
+            )
```

Comment:
If `.yaml` is present, we should just ignore `.yml`, shouldn't we?

Reply:
It's not clear that now the backend will handle that.
I understood, that now we'll send only one file.
You cannot start to send 2 files and expected that the backend process them.

Reply:
Hi Amin, I understood your reasoning and you are correct if we wanted to cover all edge cases.
But this one is covered by the sentence you added in the readme - "If both .yaml and .yml files exist in a repository, only the .yaml file will be used"

As long as we make this transparent to the user, I don't think we should tackle this scenario.